lib/battle/BattleInfo.cpp

```diff
@@ -1,1093 +1,1035 @@
-/*

```

Comment:
Whole file changed, problem with line endings?

Reply:
I made small test, by using push -f I checked for LF, CRLF, CR, but it's not resolve the problem, any ideas what it can be?